spec/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser_spec.rb

```diff
@@ -16,71 +30,60 @@
     ems
   end
 
-  let(:refresh_parser) do
-    VCR.use_cassette("#{described_class.name.underscore}_aicc") { described_class.new(ems) }
+  let(:ems_inv_to_hashes) { described_class.new(ems).ems_inv_to_hashes }
+
+  it 'will return its miq_template_type' do
+    expect(described_class.miq_template_type).to eq("ManageIQ::Providers::Lenovo::PhysicalInfraManager::Template")
   end
 
-  it 'will retrieve physical servers' do
-    result = VCR.use_cassette("#{described_class.name.underscore}_ems_inv_to_hashes") do
-      refresh_parser.ems_inv_to_hashes
+  context 'testing physical server inventory' do
```

Comment:
There is some tests not related with 'physical servers inventory' in this context ('retrieve config patterns'), I would suggest you to change the description of this context to '#ems_inv_to_hashes', once you are just testing the response of this method.